src/components/dashboard/users/SectionTitleContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent } from 'react';
+import SectionTitleButtonElement from './SectionTitleButtonElement';
+import SectionTitleLinkElement from './SectionTitleLinkElement';
+
+type IProps = {
+    title: string;
+    isBtnVisible?: boolean;
+    titleLink?: string;
+}
+
+const SectionTitleContainer: FunctionComponent<IProps> = ({title, isBtnVisible, titleLink}: IProps) => {
```

Comment:
It looks like we could default `isBtnVisible` to false since that seems to be the case for most instances.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3076/commits/9ca943fcca9c573867cf8b4ee0a9444fa790f093